src/sentry/db/models/fields/picklefield.py

```diff
@@ -1,43 +1,27 @@
-import django_picklefield
-from sentry.utils import json
+from django.db.models.fields import TextField
 
+from sentry.utils import json
 
-class PickledObjectField(django_picklefield.PickledObjectField):
-    """Sentry specific changes to the regular pickle field is the
-    changed handling for bytes (we do not allow them toplevel for
-    historic reasons) and empty strings handling.
 
-    This will eventually move over to storing JSON behind the scenes
-    and can already read JSON if it's placed in the database.  In
-    tests it will already fail with an error if code tries to put
-    values in there which cannot be serialized as JSON.
-    """
+class PickledObjectField(TextField):
+    """Actually just json"""
```

Comment:
what's troubling / annoying is we now have 4 (?) different json field types.  I'll try and reduce these once I've cut the cruft and can actually tell them apart:
- this thing
- gzippeddict (actually just json now! still need to kill the pickle bits here)
- jsonfield
- django's jsonfield


this is by far the simplest one so hopefully I can merge the other behaviours into this one.  using django's is tricky because they do some annoying bits with the json serializer that might be incompatible with what we're doing with json